app/views/simple_smart_answers/_completed_question.html.erb

```diff
@@ -1,10 +1,11 @@
-<li class="done">
-  <h3 class="question">
-    <span class="question-number"><%= completed_question_counter + 1 %></span><%= completed_question.question.title %>
```

Comment:
Would there removal of headings make it less scannable for people using screen readers to pull out the headings in the page?

Reply:
That's a good point, but I don't think that every question in this bit needs to be a heading. I've added some visually hidden headings for 'your answers' and 'current question', see new commit.